src/main/java/com/parsing/api/ExtractorLotInformation.java

```diff
@@ -0,0 +1,210 @@
+package com.parsing.api;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.parsing.model.LotId;
+import com.parsing.repository.LotIdRepository;
+import com.parsing.model.LotResult;
+import com.parsing.model.LotStatus;
+import com.parsing.model.Participant;
+import com.parsing.model.Status;
+import com.parsing.repository.LotResultRepository;
+import com.parsing.repository.ParticipantRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.client.RestTemplate;
+
+import java.math.BigDecimal;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Slf4j
+@RequiredArgsConstructor
+@Component
+public class ExtractorLotInformation {
+
+    private final String URL_LOT = "https://public.api.openprocurement.org/api/2.5/tenders/";
+    private final String LOT_STATUS = "complete";
```

Comment:
I suggest using completed status, complete is a verb and it sounds misleading for me, what do you think?